src/util/expr.h

```diff
@@ -84,6 +84,31 @@ class exprt:public irept
     return static_cast<const typet &>(find(ID_type));
   }
 
+  static void indent_str(std::string &s, unsigned indent)
+  {
+    for(unsigned i = 0; i < indent; i++)
+      s += ' ';
+  }
+
+  const std::string get_type_string(int depth) const
```

Comment:
How does this differ to just `e.type().pretty()` with the appropriate indent arguments?

Reply:
This code is now removed. Was part of the print-detailed-ir option.

Reply:
This code is removed in the next commit.

Reply:
Can you squash the removals so that this PR doesn't even touch that file?  It reduces the number of reviews / different people you need to get sign-off from.